.readthedocs.yml

```diff
@@ -23,17 +23,13 @@ build:
     - asdf plugin add uv
     - asdf install uv latest
     - asdf global uv latest
-    - uv venv --python=3.14
-
-    install:
-    - uv sync --extra docs --frozen --exact
```

Comment:
I _think_ we can drop the `uv venv` and `uv sync` commands since installing the things is done via `uvx`.